packages/minter-contracts/src/nft-contracts.ts

```diff
@@ -203,17 +192,39 @@ export async function originateEditionsNftContract(
     'editions_main',
     'fa2_multi_nft_token_editions.tz',
   );
-  const meta_uri = char2Bytes('tezos-storage:content');
-  const meta = {
-    name: 'example_name',
-    description: 'sample_token',
+
+  const editions_metadata = {
```

Comment:
Is there a way to import views from generated `editions_token_metadata_view.tz.ts` file instead of manually copying the code here?

Reply:
I actually am importing the view from the generated `editions_token_metadata_view.tz.ts` file at line 217. I couldn't find a good way to generate the `parameter` and `returnType` of the view which is what I am manually writing here.

Reply:
I see. We need to keep pushing ECAD to prioritize adding tooling support to incorporate views into TZIP-16 metadata